src/RamlToOpenApiConverter/RamlConverter.Paths.cs

```diff
@@ -30,8 +32,42 @@ private OpenApiPaths MapPaths(IDictionary<object, object> o)
             return paths;
         }
 
-        private ICollection<(OpenApiPathItem Item, string AdjustedPath)> MapPathItems(string parent, IList<OpenApiParameter> parentParameters, IDictionary<object, object> values)
+        private ICollection<(OpenApiPathItem Item, string AdjustedPath)> MapPathItems(string parent, IList<OpenApiParameter> parentParameters, IDictionary<object, object> values, IDictionary<object, object> uses)
         {
+            //replace uses in file
+            IDictionary<object, object> use_replace = new Dictionary<object, object>();
+
+            if (uses.Count > 0)
+            {
+                foreach (var path_value in values)

```

Comment:
Do not use `_` in variable names. --> This is a generic comment which applies to multiple places.

Just rename to `pathValue`

Reply:
https://github.com/StefH/RamlToOpenApiConverter/pull/18/commits/8d6029284de88e71c04103b7f0bf7eaa53ea2594